File: trunk/CreaObjetos/CreaObjetos/clases/Relacion.vb
Public Class Relacion

    Private _campo As String
    Private _Tabla As String
    Private _tipo As Integer

    Public Property Tabla() As String
        Get
            Return Me._Tabla
        End Get
        Set(ByVal value As String)
            Me._Tabla = value
        End Set
    End Property

    Public Property campo() As String
        Get
            Return Me._campo
        End Get
        Set(ByVal value As String)
            Me._campo = value
        End Set
    End Property

    Public Property tipo() As Integer
        Get
            Return Me._tipo
        End Get
        Set(ByVal value As Integer)
            Me._tipo = value
        End Set
    End Property
End Class

File: CreaObjetos/CreaObjetos.ConvertedToC#/clases/Relacion.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
namespace WindowsApplication1
{
	public class Relacion
	{

		private string _campo;
		private string _Tabla;

		private int _tipo;
		public string Tabla {
			get { return this._Tabla; }
			set { this._Tabla = value; }
		}

		public string campo {
			get { return this._campo; }
			set { this._campo = value; }
		}

		public int tipo {
			get { return this._tipo; }
			set { this._tipo = value; }
		}
	}
}
